packages/services-store/src/actions/__tests__/profileAPI.test.ts

```diff
@@ -25,45 +25,42 @@ const mockStore = configureMockStore<
 describe('Redux actions for `ProfileAPI`', () => {
   it('dispatches the action to set user authentication status', () => {
     const store = mockStore();
-    store.dispatch(setUserStatus({ user: USER_AUTHENTICATION_STATUS.AUTHENTICATED }));
+    store.dispatch(setUserStatus({ user: 'test.user@ibm.com' }));
     expect(store.getActions()).toEqual([
       {
         type: PROFILE_API_ACTION.SET_USER_STATUS,
-        status: { user: USER_AUTHENTICATION_STATUS.AUTHENTICATED },
+        status: { user: 'test.user@ibm.com' },
       },
     ]);
   });
 
-  it('dispatches the action to monitor user authentication status', () => {
-    ProfileAPI.monitorUserStatus.mockImplementation(callback => {
-      callback(null, { user: USER_AUTHENTICATION_STATUS.AUTHENTICATED });
-      callback(null, { user: USER_AUTHENTICATION_STATUS.UNAUTHENTICATED });
-    });
-    const store = mockStore();
-    store.dispatch(monitorUserStatus());
-    expect(convertValue(store.getActions())).toEqual([
-      {
-        type: PROFILE_API_ACTION.SET_USER_STATUS,
-        status: { user: USER_AUTHENTICATION_STATUS.AUTHENTICATED },
-      },
-      {
-        type: PROFILE_API_ACTION.SET_USER_STATUS,
-        status: { user: USER_AUTHENTICATION_STATUS.UNAUTHENTICATED },
-      },
-    ]);
-  });
+  // it('dispatches the action to get user authentication status', async () => {
+  //   ProfileAPI.getUserStatus.mockResolvedValue({ user: 'Unauthenticated' });
+  //   const store = mockStore();
+  //   await store.dispatch(ProfileAPI.getUserStatus());
```

Comment:
Instead of `ProfileAPI.getUserStatus()` (that makes a call to `@carbon/ibmdotcom-services` module), we need to call a Redux action here. Here is a test in this category for reference: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.14.0/packages/services-store/src/actions/__tests__/localeAPI.test.ts#L84